### PR TITLE
feat: Add Pipeline menu item behind ff

### PR DESCRIPTION
**Pull request by tiina303**
## Problem

We have a new vision for the apps ui, will start creating it behind a menu item behind a ff.
Small PRs - this just adds a menu item for me to add things to in future PRs.
Followed https://storybook.posthog.net/?path=/docs/how-to-build-a-scene--docs

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## How did you test this code?

Screenshot 2023-10-23 at 16 46 26

### CODE REVIEW

**Review by xvello — APPROVED** [review]

**Review by tomasfarias — APPROVED** [review]
I love the new icon :shipit:

File: frontend/src/lib/lemon-ui/icons/icons.tsx [hunk 1]
```diff
@@ -2383,6 +2383,17 @@ export function IconDatabase(props: LemonIconProps): JSX.Element {
     )
 }
 
+export function IconPipeline(props: LemonIconProps): JSX.Element {
```
Comment (Twixes) [inline frontend/src/lib/lemon-ui/icons/icons.tsx:2386]: Where's this icon from?

File: frontend/src/scenes/pipeline/Pipeline.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { SceneExport } from 'scenes/sceneTypes'
+import { pipelineLogic } from './pipelineLogic'
+
+export function Pipeline(): JSX.Element {
+    return (
+        // TODO: consolidate on a recommended naming convention
+        <div className="pipeline-scene">WIP: Pipeline Scene!</div>
```
Comment (Twixes) [inline frontend/src/scenes/pipeline/Pipeline.tsx:7]: Suggestion: you can start with a `PageHeader`, since that's the bare minimum of a scene

**Review by Twixes — COMMENTED** [review]
Seems alright

File: frontend/src/lib/lemon-ui/icons/icons.tsx (hunk 1 above)
Comment (tiina303) [reply frontend/src/lib/lemon-ui/icons/icons.tsx:2386]: where should we take icons from?

Comment (Twixes) [reply frontend/src/lib/lemon-ui/icons/icons.tsx:2386]: Ask @corywatilo 😄 For products (which this arguably is in some way) we should be using the same ones as on the website

**Review by Twixes — APPROVED** [review]

**Pull request merged**